addons/l10n_dk/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,152 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="account_tax_report" model="account.tax.report.line">
+        <field name="name">Tax Report - Denmark</field>
+        <field name="sequence" eval="1"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+
+    <record id="account_tax_report_line_vat_statement" model="account.tax.report.line">
+        <field name="name">Momsangivelse (positivt beløb = betale, negativt beløb = penge tilgode)</field>
+        <field name="sequence" eval="1"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+
+    <record id="account_tax_report_line_vat_statement_sales_group" model="account.tax.report.line">
+        <field name="name">Salgsmoms</field>
+        <field name="code">Momsangivelse</field>
+        <field name="sequence" eval="1"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_vat_statement"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_sales_tax" model="account.tax.report.line">
+        <field name="name">Salgsmoms (udgående moms)</field>
+        <field name="tag_name">Salgsmoms (udgående moms)</field>
+        <field name="sequence" eval="1"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_vat_statement_sales_group"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_international_purchase_products" model="account.tax.report.line">
+        <field name="name">Moms af varekøb i udlandet (både EU og lande uden for EU)</field>
+        <field name="tag_name">Moms af varekøb i udlandet (både EU og lande uden for EU)</field>
+        <field name="sequence" eval="2"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_vat_statement_sales_group"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_international_purchase_services" model="account.tax.report.line">
+        <field name="name">	Moms af ydelseskøb i udlandet med omvendt betalingspligt</field>
```

Comment:
Not sure what you want to achieve with the trailing space...

Reply:
Strange. Must have happened when I copied it in from the runbot on where we configured it prior to making the code change. It is fixed now